docs/how-to-guides/block-tutorial/extending-the-query-loop-block.md

```diff
@@ -0,0 +1,195 @@
+
+We need a way to tell the editor that this block is indeed your specific variation. This is what the `isActive` property is made for: it's a way to determine whether a certain variation is active based on the block's attributes. You might be tempted to then do something like this:
+
+```js
+{
+	/** ...variation properties */
+	isActive: [ 'postType' ],
+}
+```
+
+In this way, Gutenberg will recognize the block as your variation any time the `postType` matches `book`! That's awesome, but the problem is that now Gutenberg will recognize the block as your specific variation any time the `postType` is set to `book`, which is not what we want: other plugins might want to publish variations based on the `book` post type, or we might just not want the variation to be recognized every time the user sets the type to `book` manually through the editor settings.
+
+That's why the Query Loop block exposes a special attribute called `namespace`: it really doesn't do anything inside the block implementation, and it's used as an easy way for extenders to recognize and scope their own variation. So you would use it like so:
+
+```js
+{
+	/** ...variation properties */
+	attributes: {
+		/** ...variation attributes */
+		namespace: 'my-plugin/books-list',
+	},
+	isActive: [ 'postType', 'namespace' ],
+}
+```
+
+Like so, Gutenberg will know that it is your specific variation only in the case it matches your custom namespace! So convenient!
+
+## Extending the query
+
+Even with all of this, your custom post type might have unique requirements: it might support certain custom attributes that you might want to filter and query for, or some other query parameters might be irrelevant or even completely unsupported! We have build the Query Loop block with such use-cases in mind, so let's see how you can solve this problem.
+
+### Disabling irrelevant or unsupported query controls
+
+Let's say you don't use at all the `sticky` attribute in your books, so that would be totally irrelevant to the customization of your block. In order to not confuse the users as to what a setting might do, and only exposing a clear UX to them, we want this control to be unavailable. Furthermore, let's say that you don't use the `author` field at all, which generally indicates the person who has added that post to the database, instead you use a custom `bookAuthor` field. As such, not only keeping the `author` filter would be confusing, it would outright break your query.
+
+For this reason, the Query Loop block supports a property called `allowedControls` which accepts an array of keys of the controls we want to whitelist. By default, we accept all the controls, but as soon as we provide an array to this property, we want to be specific and whitelist only the controls which are going to be relevant for us!
+
+As of version 13.9, the following controls are available:
```

Comment:
Should we add the GB version or the WP one here ? 🤔

Reply:
We can leave this discussion open for documentation folks to chime in. In general we will need some other small changes in some other files for the documentation to work, but we can add them in the end, when the copy is finalized.